hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -358,12 +358,12 @@ public final class HoodieMetadataConfig extends HoodieConfig {
       .sinceVersion("1.0.0")
       .withDocumentation("Parallelism to use, when generating partition stats index.");
 
-  public static final ConfigProperty<Boolean> ENABLE_PARTITION_STATS_INDEX_TIGHT_BOUND = ConfigProperty
-      .key(METADATA_PREFIX + ".index.partition.stats.tightBound.enable")
-      .defaultValue(true)
+  public static final ConfigProperty<Boolean> PARTITION_STATS_INDEX_TIGHT_BOUND_EVERY_COMMIT = ConfigProperty
```

Comment:
thinking if we should we name this based on CAP theorem. eventually consistent or strongly consistent. 

hoodie.metadata.index.partition.stats.enable.strongly.consistent.stats

Reply:
synced up w/ ethan. 
may be 
```
hoodie.metadata.index.partition.stats.recompute.on.every.write
```

Reply:
Actually, the config is too much of a mouthful. I liked tightBound better, and "tight bound" is a standard mathematical phrase when using ranges. But, I'll leave it upto you guys.

Reply:
Unrelated but we need to use all small letters in the config name. The config was not getting picked up when there was a capital B in the config name - `tightBound`. Just mentioning in case we are changing it.